src/main/java/com/jalasoft/jfc/controller/UserController.java

```diff
@@ -148,10 +157,11 @@
     @ApiOperation(value = "Authorization", notes = "Delete user values by Id",
             response = Response.class)
     public ResponseEntity<?> deleteById(@RequestParam("Id") int id) {
```

Comment:
It is necessary?

Reply:
yes, because it is part of CRUD operations.